users_database.py

```diff
@@ -0,0 +1,95 @@
+import sqlite3
+import random
+import string
+
+def get_random_email():
+    letters = string.ascii_lowercase
+    emails = ["gmail.com", "hotmail.com", "aol.com", "yahoo.com", "test.org"]
+    email = (''.join((random.choice(letters)) for i in range(random.randint(10, 20)))) + "@" + random.choice(emails)
+    return email
+class UsersDatabase:
+    def __init__(self):
+        self.connection = None
+        self.db = None
+        self.user = None
+        try:
+            self.connection = sqlite3.connect("./databases/users.db")
+            self.db = self.connection.cursor()
+            self.db.execute('''
+                            CREATE TABLE users (
+                                user_id INTEGER PRIMARY KEY AUTOINCREMENT,
+                                email TEXT,
+                                role TEXT CHECK( role IN ('admin', 'general', 'investigator') ) NOT NULL DEFAULT 'general',
+                                password TEXT
+                            );
+                            ''')
+            self.connection.commit()
+        except sqlite3.OperationalError:
+            pass
+        except sqlite3.Error as e:
+            print(e)
+            
+    def add_user(self, email: str, role: str = None):
+        if role:
+            self.db.execute( "INSERT INTO users (email, role, password) VALUES (?, ?, ?);", (email, role, "password") )
+        else:
+            self.db.execute( "INSERT INTO users (email, password) VALUES (?, ?);", (email, "password") )
+        self.connection.commit()
+        
+    def get_all_users(self):
+        self.db.execute("SELECT * FROM users")
+        return self.db.fetchall()
+        
+    def get_user_from_id(self, id: int):
+        self.db.execute("SELECT * FROM users WHERE user_id = ?", (id,) )
+        return self.db.fetchall()[0]
```

Comment:
need to handle the scenario when this function gets triggered with an id which is not yet registered with us.